src/index.html

```diff
@@ -8,6 +8,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+    </ul>
+    <ul class="stars stars--1">
```

Comment:
Add empty line between multiline siblings [read](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)
Apply all over the page
```suggestion
    </ul>
    
    <ul class="stars stars--1">
```